render-superhero.js

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Get in the practice of adopting consistent and clear naming conventions in your file structure. Imagine someone completely new to your codebase coming in and trying to pair up which *-data.js files work with which render-*.js files. Make 'em match!